space-cli/cmd/utils/paths.go

```diff
@@ -50,3 +51,51 @@ func getHomeDirectory() string {
 func getAccountConfigPath() string {
 	return fmt.Sprintf("%s/accounts.yaml", GetSpaceCloudDirectory())
 }
+
+// GetMountHostsFilePath returns the path of the hosts files to be mounted in in space cloud
+func GetMountHostsFilePath() string {
+	if runtime.GOOS == "windows" {
+		return fmt.Sprintf("%s/.space-cloud/hosts", getHomeDirectoryToolBox())
+	}
+	return GetSpaceCloudHostsFilePath()
+}
```

Comment:
do we need to create another function?  just do the check for windows in GetSpaceCloudHostsFilePath() itself. Do this for all the function.